packages/react/src/wishlists/hooks/useWishlistSets.ts

```diff
@@ -0,0 +1,103 @@
+/**
+ * Hook to provide data for the business logic attached to the wishlist sets.
+ *
+ * @module useWishlistSets
+ * @category Wishlist Sets
+ * @subcategory Hooks
+ */
+import {
+  areWishlistSetsLoading,
+  areWishlistSetsWithAnyError,
+  fetchWishlistSets as fetchWishlistSetsAction,
+  getAllWishlistSetsErrors,
+  getWishlistSets,
+  getWishlistSetsError,
+  isAnyWishlistSetLoading as isAnyWishlistSetLoadingSelector,
+  resetWishlistSets as resetWishlistSetsAction,
+  resetWishlistSetsState as resetWishlistSetsStateAction,
+} from '@farfetch/blackout-redux/wishlists';
+import { useAction } from '../../helpers';
+import { useSelector } from 'react-redux';
+import type { UseWishlistSets } from './types';
+
+/**
+ * Provides Redux actions and state access for dealing with
+ * wishlist sets business logic.
+ *
+ * @memberof module:wishlists/hooks
+ *
+ * @returns {object} All the handlers, state, actions and relevant data needed
+ * to manage any wishlist operation.
+ */
+const useWishlistSets: UseWishlistSets = () => {
+  // Selectors
+  const allWishlistSetsErrors = useSelector(getAllWishlistSetsErrors);
+  const areLoading = useSelector(areWishlistSetsLoading);
+  const error = useSelector(getWishlistSetsError);
+  const isAnyWishlistSetLoading = useSelector(isAnyWishlistSetLoadingSelector);
+  const isAnyWishlistSetWithError = useSelector(areWishlistSetsWithAnyError);
+  const wishlistSets = useSelector(getWishlistSets);
+  // Actions
+  const fetchWishlistSets = useAction(fetchWishlistSetsAction);
+  const resetWishlistSets = useAction(resetWishlistSetsAction);
+  const resetWishlistSetsState = useAction(resetWishlistSetsStateAction);
+
+  return {
+    /**
+     * List of errors states for the wishlist sets.
```

Comment:
```suggestion
     * List of error states for the wishlist sets.
```

I'm not quite sure e about this one but I share it anyway 😄

Reply:
I think the way you suggested makes more sense 🙂